projects/core/src/index.ts

```diff
@@ -1,11 +1,13 @@
 export {MASKITO_DEFAULT_OPTIONS} from './lib/constants';
 export {Maskito} from './lib/mask';
 export {
+    AsyncMaskitoElementPredicate,
     MaskitoElementPredicate,
     MaskitoMask,
     MaskitoOptions,
     MaskitoPlugin,
     MaskitoPostprocessor,
     MaskitoPreprocessor,
+    SyncMaskitoElementPredicate,
```

Comment:
All public API should be start from `maskito` / `Maskito` prefix. 

Let's rename new types:
```
SyncMaskitoElementPredicate => MaskitoElementPredicateSync
AsyncMaskitoElementPredicate => MaskitoElementPredicateAsync
```